### ISSUE TITLE
Accessibility - Provide way to customize aria-label

### ISSUE THREAD

**Issue by antoniolucasnobar** [richardtallent/vue-simple-calendar#257]
The Calendar (https://github.com/richardtallent/vue-simple-calendar/blob/main/src/CalendarView.vue#L3) and the Header buttons (https://github.com/richardtallent/vue-simple-calendar/blob/main/src/CalendarViewHeader.vue#L4-L20) have fixed aria-label in english. This way visually impaired users are unable to understand what button does (I am from Brazil).

Ideally, the aria-label should adhere to the chosen locale. Or, at least provide bindings to customize these fields.

Two pictures below are prints from the links above.

<img width="440" height="200" alt="Image" src="https://github.com/user-attachments/assets/2a7f5fa2-8fb6-4421-a33b-42033b309003" />

<img width="926" height="550" alt="Image" src="https://github.com/user-attachments/assets/04a389c3-cf8d-4987-8cc2-1ecbfbe9592a" />

Please let me know if you need any information.

**Comment by richardtallent** [comment]
Hi! Since browsers don't have any built-in localized dictionary of suitable terms, it sounds like a new prop would be needed to allow the user to provide localized support.

I don't speak Portuguese, only a little Spanish, but if you provide a list of the labels I'm using currently in English and the appropriate terms in Portuguese, I'd be happy to add it as a second language with built-in support.

**Comment by antoniolucasnobar** [comment]
Hello Richard, thanks for your kind message. I am so sorry I missed. I put the portuguese terms below each aria-label I found

```
aria-label="Calendar" 
Calendário
aria-label="Previous Period"
Período Anterior
aria-label="Current Period"
Período Atual
aria-label="Next Period"
Próximo Período
aria-label="Previous Year"
Ano Anterior
aria-label="Next Year"
Próximo Ano
```


There is another one,
```
:aria-label="day.getDate().toString()"
"Fri Dec 19 2025 14:38:17 GMT-0300 (Horário Padrão de Brasília)" 
```
toString() always returns the day in english, regardless the system language. 

One idea is to use  
```
:aria-label="day.getDate().toLocaleString(displayLocale)"
"19/12/2025, 14:37:56" 
```
The output is different, if you want to keep the same information, there is a longer option
```
:aria-label="day.getDate().toLocaleString(displayLocale, {
  weekday: 'short',
  year: 'numeric',
  month: 'short',
  day: 'numeric',
  hour: '2-digit',
  minute: '2-digit',
  second: '2-digit',
  timeZoneName: 'long'
});
"sex., 19 de dez. de 2025, 14:47:11 Horário Padrão de Brasília" 
```

Thank you once again.
Best regards!
